src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/VariantLoaderStepTest.java

```diff
@@ -56,6 +57,7 @@
  * Test for {@link VariantLoaderStep}
  */
 @RunWith(SpringRunner.class)
+@ActiveProfiles("variant-annotation-mongo")
```

Comment:
This class has nothing to do with the annotation, is it still necessary to enable that sub-profile for this to work?